app/controllers/api/v1/saved_scenarios_controller.rb

```diff
@@ -18,7 +18,14 @@ def index
       end
 
       def show
-        render json: current_user.saved_scenarios.find(params.require(:id))
+        saved_scenario = current_user
+          .saved_scenarios
+          .includes(:saved_scenario_users, :users)
+          .find(params.require(:id))
+
+        render json: saved_scenario.as_json.merge(
+          "saved_scenario_users" => saved_scenario.saved_scenario_users.as_json
```

Comment:
We are lenient on providing a lot of info on the users, even if we don't know who is asking this information.
Let's anonimise this information: leave out email adresses, names, anything we don't need!